package.json

```diff
@@ -61,14 +61,13 @@
     "i18n:checks": "./lighthouse-core/scripts/i18n/assert-strings-collected.sh",
     "i18n:collect-strings": "node lighthouse-core/scripts/i18n/collect-strings.js",
     "update:sample-artifacts": "node lighthouse-core/scripts/update-report-fixtures.js -G",
-    "update:sample-json": "yarn i18n:collect-strings && node ./lighthouse-cli -A=./lighthouse-core/test/results/artifacts --throttling-method=devtools --output=json --output-path=./lighthouse-core/test/results/sample_v2.json && node lighthouse-core/scripts/cleanup-LHR-for-diff.js ./lighthouse-core/test/results/sample_v2.json --only-remove-timing",
+    "update:sample-json": "yarn i18n:collect-strings && node ./lighthouse-cli -A=./lighthouse-core/test/results/artifacts --throttling-method=devtools --output=json --output-path=./lighthouse-core/test/results/sample_v2.json && node lighthouse-core/scripts/cleanup-LHR-for-diff.js ./lighthouse-core/test/results/sample_v2.json --only-remove-timing && yarn compile-proto && yarn build-proto-roundtrip",
```

Comment:
that's quite the line these days 😆 

just thinking out loud below 💭 

I guess the contributor thinking here is that if they already had to run `update:sample-json` then they would've failed the build without `compile-proto` so forcing protobuf in the path of this command isn't really a barrier. If they just needed string collection that would still run before all this junk failed anyhow. 👍

Reply:
^ yes.  any meaningful changes require running both. 

--------

separately, can we bikeshed the script name?? :) how about `update:goldens`?